src/mdx-pages/guides/audio-tracks.mdx

```diff
@@ -0,0 +1,165 @@
+---
+title: Audio Tracks
+category: tracks
+order: 2
```

Comment:
The frontmatter for guides affects the grouping on the guides landing page and the order within each group. Also, `order` is optional - sorting will fall back to `title`.